### ISSUE TITLE
cockpit ui is not compatible with the phone

### ISSUE THREAD

**Issue by jake2009** [cockpit-project/cockpit#4045]
I try to upload images but fail
thank  rhwood to upload this images

**Comment by stefwalter** [comment]
Doesn't look like the screenshots uploaded. Could you try again?

**Comment by rhwood** [comment]
The login screen is fine: ![img_3515](https://cloud.githubusercontent.com/assets/297232/13980894/64252cba-f0b7-11e5-94d1-3e4f81976076.jpg)

System is unreadable after logging in (100% zoom): ![img_3516](https://cloud.githubusercontent.com/assets/297232/13980897/6473dfb8-f0b7-11e5-9974-de38e898af79.jpg)

Zoom all the way out, and its clear that something is not resizing for the screen width: ![img_3517](https://cloud.githubusercontent.com/assets/297232/13980896/646b3ce6-f0b7-11e5-98f7-d1466a0047ae.jpg)

Double tapping the Containers so the content fits suggests the Containers would be usable if the menu wasn't fixed to the side: ![img_3518](https://cloud.githubusercontent.com/assets/297232/13980895/645bf4ac-f0b7-11e5-8064-27532f25ef15.jpg)

All images are from an iPhone 5s and are reduced to 50% of original size. Cockpit 0.93 from CentOS extras yum repo.

**Comment by dcrdev** [comment]
Also as this is in relation to iOS, the terminal is incompatible as well in that it's impossible to trigger the keyboard to pop up.

**Comment by jake2009** [comment]
I only think the login screen is fine。Else is not fine to phone 。

**Comment by andreasn** [comment]
The first biggest blocker for this seems to be the sidebar, so I'll focus on that first.

**Comment by garrett** [comment]
I happened upon this issue today. It was addressed a while ago in https://github.com/cockpit-project/cockpit/pull/7482, which redid the navigation and added a mobile mode.

Some parts of Cockpit are a bit wonky or problematic in mobile (especially big table-based lists), but most of it works. We're always working to improve mobile too. For example, the new form layout code that we've been porting to is not just great for desktop; it's also automatically responsive and forms adapt to smaller screens.
